src/umpire/op/CudaGetAttributeOperation.cpp

```diff
@@ -21,7 +21,8 @@ bool CudaGetAttributeOperation<ATTRIBUTE>::check_apply(void* src_ptr, umpire::ut
   error = ::cudaGetDeviceProperties(&properties, 0);
 
   if (error != cudaSuccess) {
-    UMPIRE_ERROR("cudaGetDeviceProperties( device = " << 0 << "),"
+    UMPIRE_ERROR(runtime_error,
+                 "cudaGetDeviceProperties( device = " << 0 << "),"
```

Comment:
Shouldn't this be converted to use `fmt`?

Reply:
Yes, I must have missed it.